cmd/utils/shutdown/run.go

```diff
@@ -0,0 +1,22 @@
+package shutdown
+
+import "errors"
+
+func RunThenTrigger(trigger Trigger, triggerOnSuccess bool, fn func() error) {
```

Comment:
```suggestion
func RunThenTrigger(trigger Trigger, alwaysTrigger bool, fn func() error) {
```

Slightly confusing, as it's an effectively unconditional call of trigger.

Reply:
I find `alwaysTrigger=false` also confusing. What about reverting it as in `skipOnSuccess`?

Reply:
Works for me